Disable LMS service by default

sfonn opened the pull request:
#### Merge Checklist  <!-- REQUIRED -->
**All** boxes should be checked before merging the PR
- [x] The changes in the PR have been built and tested
- [x] cgmanifest file has been updated if required
- [x] Ready to merge

#### Description <!-- REQUIRED -->
<!-- Please include a summary of the changes and the related issue. List any dependencies that are required for this change. -->
Disable and mask LMS service by default so that it does not cause systemd to wait for /dev/mei device to appear after boot by default which causes delay in boot time on platforms which do not have mei.

<!-- Fixes # (issue) -->

#### Any Newly Introduced Dependencies
<!-- Please describe any newly introduced 3rd party dependencies in this change. List their name, license information and how they are used in the project. -->

#### How Has This Been Tested? <!-- REQUIRED -->
<!-- Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration. -->
Build edge-image-dev.json and verified boot in Xeon platform (which does not have mei). Before change, boot is 40+ sec. After change:
1.715s (loader) + 2.877s (kernel) + 5.878s (initrd) + 2.773s (userspace)
tested on dut with amt vpro that after enable lms.service manually, "sudo rpc amtinfo" returns expected info and lms service started successfully.

andy-vm commented on SPECS/intel-lms/intel-lms.spec:
```diff
@@ -57,6 +57,11 @@ cp -r Docs/Licenses/* %{buildroot}%{_datadir}/licenses/%{name}
 %post -f UNS/linux_scripts/post.rpm
+# Disable lms service by default
+systemctl disable lms.service > /dev/null 2>&1 || :
+systemctl stop lms.service > /dev/null 2>&1 || :
```
this line is not required? service is not running

sfonn replied:
vpro team will take necessary actions to enable only for nodes which support vpro amt feature so that boot time on non-supporting edge nodes do not boot delay due to waiting for non-existent hw.

anujm1 commented on SPECS/intel-lms/intel-lms.spec:
```diff
@@ -57,6 +57,11 @@ cp -r Docs/Licenses/* %{buildroot}%{_datadir}/licenses/%{name}
 %post -f UNS/linux_scripts/post.rpm
+# Disable lms service by default
+systemctl disable lms.service > /dev/null 2>&1 || :
+systemctl stop lms.service > /dev/null 2>&1 || :
```
I just noticed that we're invoking `post.rpm` which is starting this service. Can we stop doing that?

sfonn replied:
okay, will work on this.

sfonn replied:
modified to not call post.rpm in spec file and instead:
- use systemd-rpm macro to handle lms.service
- only perform rsyslog and udevadm actions if installation on actual system instead of chroot context.

sfonn committed:
Update intel-lms post rpm install hook action

Update intel-lms post rpm install hook actions in spec
file instead of calling script file.
LMS service is not enabled by default.

Signed-off-by: Swee Yee Fonn <swee.yee.fonn@intel.com>

anujm1 commented on SPECS/intel-lms/intel-lms.spec:
```diff
@@ -75,6 +85,9 @@ rm -rf %{buildroot}%{_docdir}/lms
 
 
 %changelog
+* Mon Jun 23 2025 Swee Yee Fonn <swee.yee.fonn@intel.com> - 2506.0.0.0-3
```
`Release` should be bumped as well
